usaspending_api/references/management/commands/load_reference_data.py

```diff
@@ -29,7 +29,7 @@ def handle(self, *args, **options):
             call_command('loadtas', 'gtas-sf133')
 
         self.logger.info("Loading program_activity.csv")
-        call_command('loadprogramactivity', 'usaspending_api/data/program_activity.csv')
+        call_command('loadprogramactivity')
```

Comment:
We should do a check to see if this is running locally, similar to the TAS loader above.

Reply:
Agreed. Fixing.